model-optimizer/extensions/middle/quantize_linear_resolver.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.middle.MulFakeQuantizeFuse import MulFakeQuantizeFuse
+from extensions.middle.ReluQuantizeFuse import ReluFakeQuantizeMark
+from extensions.ops.Cast import Cast
+from extensions.ops.elementwise import Mul
+from extensions.ops.fakequantize import FakeQuantize
+from mo.front.common.partial_infer.utils import float_array, int64_array
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, rename_nodes
+from mo.middle.replacement import MiddleReplacementPattern
+from mo.ops.const import Const
+from mo.ops.reshape import Reshape
+from mo.utils.error import Error
+
+
+class QuantizeLinearResolver(MiddleReplacementPattern):
+    """
+    Replaces QuantizeLinear with FakeQuantize
+    Transformation result depend on from axis value.
+    If axis not set or default value equal 1 QuantizeLinear can be replace with the following subgruph:
+        QuantizeLinear -> FakeQuantize(input
+                                       Mul(y_scale, Const(low_value))
+                                       Mul(y_scale, Const(high_value))
+                                       Const(low_value)
+                                       Const(high_value))
+        low_value and high_value depend on from y_zero_point type
+    In other cases y_scale and y_zero_point with addition broadcasting.
+    Target shape for y_scale and y_zero_point depend on axis value.
+    """
+    enabled = True
+
+    def run_after(self):
+         return [MulFakeQuantizeFuse, ReluFakeQuantizeMark]
```

Comment:
Why? The `MulFakeQuantizeFuse` transformation is disabled and executed manually in the fusings transformation so there is no any sense to add dependency to it. 

Also the `QuantizeLinearResolver` should be executed before `ReluQuantizeFuse` because if you do vice versa then you miss the opportunity to fuse decomposed FQ from the QuantizeLinear with Relu.

Reply:
When I try to call QuantizeLinearResolver  before ReluQuantizeFuse  I get following exception: networkx.exception.NetworkXError: Input is not a valid edge list

Reply:
And what is the reason of `networkx.exception.NetworkXError: Input is not a valid edge list` exception? May be there is some bug. Please, analyse.

Reply:
![sort_replasment2](https://user-images.githubusercontent.com/29801542/118371043-52ed7800-b5b3-11eb-82b7-0184fa559c74.PNG)
QuantizeLinerResolver will be call before FackeQuantizeFuse. 
According to rules of order replacements transformations will be call in following order: QuantizeLinerResolver -> FackeQuantizeFuse -> ReluQuantizeFuse